app.cr

```diff
@@ -34,12 +34,13 @@ end
 
 def fetch_filter(context)
   filter = context.params["filter"]? || ""
-  filter.strip
+  filter = filter.strip.downcase unless filter.empty?
+  filter
 end
 
 private def matches_filter?(item: GithubRepo, filter: String)
-  item.name.includes?(filter) ||
-    (item.description && item.description.not_nil!.includes?(filter))
+  item.name.downcase.includes?(filter) ||
```

Comment:
First that comes to the mind is to use `try` also:
```ruby
private def matches_filter?(item: GithubRepo, filter: String)
  item.name.downcase.includes?(filter) ||
    item.description.try(&.downcase.includes? filter)
end
```

It will look a bit shorter, but not sure that this is the real improvement. Your thoughts ?